sap-labs-france/ev-dashboard
# Use average per asset per minute before summing to site area level while displaying site area chart component

Test cases: 
- Check if the site area consumption chart aggregates per minute per asset before summing up to site area aggregation. 
  1. Manually add additional consumption to a site area asset for a given minute in Consumptions.
  2. Check if the average of the instantWatts is calculated for the given minute instead of summing the instantWatts within a minute.
- Check if the axis on site area consumption chart shows 'Consumption by Site Area'
- Check if the axis on site area consumption chart shows the values with 'A' or 'kW'
- Check if the negative axis is shown indicating production values.

Fixes:
- Uses non facet, only grouping based mongo db aggregation pipeline methods to aggregate.

Sample:
![image](https://user-images.githubusercontent.com/5779016/149799706-5ef30337-e0b6-4a2d-be40-aeed2c19a2ce.png)

[ev-server changes](https://github.com/sap-labs-france/ev-server/pull/3245)
